src/components/RSVPform.js

```diff
@@ -0,0 +1,175 @@
+import React, {useState} from "react";
+import {Form, Button, Col, FormControl, InputGroup} from "react-bootstrap";
+import {db} from "../firebase";
+
+const RSVPform = () => {
+  const [firstName, setFirstName] = useState("");
+  const [lastName, setLastName] = useState("");
+  const [rsvp, setRsvp] = useState("");
+  const [plusOne, setPlusOne] = useState(false);
+  const [plusFName, setPlusFName] = useState("");
+  const [plusLName, setPlusLName] = useState("");
+  const [allergies, setAllergies] = useState("");
+  const [guestEmail, setEmail] = useState("");
+
+  const addUser = () => {
+    const data = {
+      firstName: firstName,
+      lastName: lastName,
+      uid: new Date().getTime(),
+      rsvp: rsvp,
+      plusOne: plusOne,
+      plusFName: plusFName,
+      plusLName: plusLName,
+      allergies: allergies,
+      email: guestEmail,
+    };
+    db.collection("guests")
+      .doc(data.uid.toString())
+      .set(data)
+      .then(() => {
```

Comment:
Would it be more convenient to use `async/await`?

Reply:
Yep, totally. I was following existing tutorial and didn't think too much about it. Thanks for reminding me!